CHANGELOG.asciidoc

```diff
@@ -7,6 +7,52 @@
 === Beats version 8.4.1
 https://github.com/elastic/beats/compare/v8.4.0\...v8.4.1[View commits]
 
+==== Known Issue 
+
+// tag::credentials-error[]
+
+*Filebeat*
+
+Filebeat agents configured to read from S3 or SQS may return the following
```

Comment:
```suggestion
Filebeat agents configured to read from S3 input or S3 input with SQS notification setup may return the following
```

Reply:
I applied Andrea's recommendation to make this more generic instead.